### PR TITLE
[Backend] - Roles Integration tests. Initial commit

### PR DESCRIPTION
**Pull request by Chismur**

### CODE REVIEW

File: odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/RoleRepository.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import org.opendatadiscovery.oddplatform.model.tables.pojos.RolePojo;
 
 public interface RoleRepository extends CRUDRepository<RolePojo> {
+
```
Comment (damirabdul) [inline odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/RoleRepository.java:7]: why?

**Review by damirabdul — COMMENTED** [review]
Please use assertj for assertions

File: odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/RoleRepository.java (hunk 1 above)
Comment (Chismur) [reply odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/RoleRepository.java:7]: There is no strict checkstyle, with this whitespace looks more nice

Comment (damirabdul) [reply odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/RoleRepository.java:7]: we have the same style in all files, let's not change it for 1 file

File: odd-platform-api/src/test/java/org/opendatadiscovery/oddplatform/repository/RolesRepositoryImplTest.java [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    @Test
+    @DisplayName("Creates role pojo, expecting role pojo in db")
+    void testCreatesRolePojo() {
+        final RolePojo rolePojo = new RolePojo().setName(UUID.randomUUID().toString());
+
+        final RolePojo actualRolePojo = roleRepository.create(rolePojo);
+
+        assertThat(actualRolePojo).isNotNull();
+        assertThat(actualRolePojo.getId()).isNotNull();
+        assertThat(actualRolePojo.getName()).isNotNull();
```
Comment (damirabdul) [inline odd-platform-api/src/test/java/org/opendatadiscovery/oddplatform/repository/RolesRepositoryImplTest.java:31]: may be better to compare that name is equals to rolePojo.getName() ?

**Review by damirabdul — APPROVED** [review]

**Pull request merged**